src/PIL/ImageFont.py

```diff
@@ -56,7 +61,7 @@ class Layout(IntEnum):
     core = DeferredError.new(ex)
 
 
-def _string_length_check(text):
+def _string_length_check(text: str | bytes) -> None:
```

Comment:
```suggestion
def _string_length_check(text: str) -> None:
```
This is an internal method that will only be passed strings when it is called elsewhere in this file.